addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -459,6 +460,22 @@ private boolean checkHyperGraphQlEngine() {
                 "Validation error of type UnknownDirective: Unknown directive deprecated @ '__typename'");
     }
 
+    private boolean checkInigoEngine() {
```

Comment:
Where's the source link?

Reply:
The project doesn't see to be open source. Do you want me to link/comment the graphw00f PR?

Reply:
Added